vip_spec.xsd

```diff
@@ -160,7 +160,14 @@ reference elements for polling location
 
   <xs:complexType name="BallotStyle">
     <xs:sequence>
-      <xs:element name="OrderedContest" type="OrderedContest" minOccurs="0" maxOccurs="unbounded" />
+      <xs:element name="OrderedContest" minOccurs="0" maxOccurs="unbounded">
```

Comment:
OrderedContest should not be inlined, it should have an ID added to it. Otherwise with **N** different ballot styles, you'll be repeating the Presidential, Gubernatorial, Attorney General, etc, races **N** times. OrderedContest should be a stand-alone item with an ID and should be referred to by the ID.

I'm not sure how this slipped through in previous iterations, but reference-by-ID is the right way to go.